net/interfaces/interfaces_test.go

```diff
@@ -46,22 +46,22 @@ func TestLikelyHomeRouterIP(t *testing.T) {
 	t.Logf("myIP = %v; gw = %v", my, gw)
 }
 
-func TestIsGlobalV6(t *testing.T) {
+func TestIsUsableV6(t *testing.T) {
 	tests := []struct {
 		name string
 		ip   string
 		want bool
 	}{
 		{"first ULA", "fc00::1", true},
-		{"Tailscale", "fd7a:115c:a1e0::1", false},
+		{"Tailscale", "fd7a:115c:a1e0::1", true},
 		{"Cloud Run", "fddf:3978:feb1:d745::1", true},
 		{"zeros", "0000:0000:0000:0000:0000:0000:0000:0000", false},
 		{"Link Local", "fe80::1", false},
 		{"Global", "2602::1", true},
 	}
 
 	for _, test := range tests {
-		if got := isGlobalV6(netaddr.MustParseIP(test.ip)); got != test.want {
+		if got := isUsableV6(netaddr.MustParseIP(test.ip)); got != test.want {
 			t.Errorf("isGlobalV6(%s) = %v, want %v", test.name, got, test.want)
```

Comment:
s/isGlobalV6/isUsableV6/

Reply:
Still there as of d3ed7ccba3b209513b7e08875b4ec923faa4874b (the t.Errorf line's `isGlobalV6`)